docs/OV_Runtime_UG/CpuPluginOptimizations.md

```diff
@@ -0,0 +1,81 @@
+# Internal CPU Plugin Optimizations {#openvino_docs_IE_DG_CpuPluginOptimizations}
```

Comment:
who is interested in it?

Reply:
As far as I know this section was created for the WB team. But now it was decided to remove this part from the documentation and move it to some more suitable place.